ide/src/main/java/com/devonfw/tools/ide/url/updater/graalvm/GraalVmUrlUpdater.java

```diff
@@ -21,13 +22,20 @@ protected String getVersionPrefixToRemove() {
 
   @Override
   protected void addVersion(UrlVersion urlVersion) {
-
-    String baseUrl = "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${version}/graalvm-ce-java11-";
-    doAddVersion(urlVersion, baseUrl + "windows-amd64-${version}.zip", WINDOWS, X64);
-    doAddVersion(urlVersion, baseUrl + "linux-amd64-${version}.tar.gz", LINUX, X64);
-    doAddVersion(urlVersion, baseUrl + "darwin-amd64-${version}.tar.gz", MAC, X64);
-    doAddVersion(urlVersion, baseUrl + "darwin-aarch64-${version}.tar.gz", MAC, ARM64);
-
+    // if (urlVersion.getName().startWith("jdk")
```

Comment:
What is the best condition to use here? Is not it easier to use the `urlVersion.getName()` method?

Reply:
Yes that would be better. You could also try something like `urlVersion.getStart().getLettersString()` to get the potential `jdk` prefix but the out-commented code is maybe the one most easy to understand by different developers.

Reply:
Done